gulpfile.js/tasks/styles.js

```diff
@@ -30,8 +31,9 @@ gulp.task('styles:sass', function () {
         .pipe(sass({
             errLogToConsole: true,
             includePaths: includePaths,
-            outputStyle: 'expanded'
+            outputStyle: 'compressed'
         }))
+        .pipe(cssnano())
```

Comment:
Source maps would be excellent! Personally I leave source maps in when I deploy things because it means you get nicer debugging in the wild for free - which whilst isn't ideal is something you have to do from time to time. Sourcemaps aren't loaded unless dev tools are open so there's no overhead :)